js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,237 @@
+import React, { Fragment } from "react";
+import styled from "styled-components";
+import { Select } from "@yoast/components/src/select/Select";
```

Comment:
We should not load in components directly like this, but expose them via the `index.js`.
```suggestion
import { Select } from "@yoast/components";
```

Reply:
Nice catch!